lib/worker.js

```diff
@@ -225,7 +249,11 @@ class AssetComputeWorker {
         try {
             console.log(`generating all ${this.renditions.length} renditions...`);
             for (const rendition of this.renditions) {
-                console.log(rendition.instructionsForEvent());
+                setConsoleLogPrefix(`[rendition ${rendition.index}]`);
+                await this.preparePostProcess(rendition);
```

Comment:
try/catch around it. In case it ever crashes, to restore the proper console in `finally`.

Reply:
Ack - same as done in `compute()`.

I was wondering if there would be any less intrusive way to (temporarily) change the log prefix, including restoring on errors. It just adds more boilerplate to the code right now.